README.md

```diff
@@ -23,6 +23,15 @@ The infrastructure, deployment, and operational parts of this project are identi
 
 > _Note_: When packaging/deploying the serverless application you'll not only need the normal `STAGE`, etc. environment variables in that guide but will also need a `API_KEY_SECRET` value.
 
+## Dependencies
+
+This project consumes `@formidablelabs/dogs` from [GitHub Package Registry](https://github.com/features/packages) which requires the following modification to your `.npmrc` file.
+
+```sh
+echo '@formidablelabs:registry=https://npm.pkg.github.com/' > ~/.npmrc
+echo //npm.pkg.github.com/:_authToken=\${GITHUB_PAT} >> ~/.npmrc
```

Comment:
What is `GITHUB_PAT` and is it supposed to be in the environment?

Reply:
I've included a description for `GITHUB_PAT` in the newest commit.